phoenix-core/src/main/java/org/apache/phoenix/execute/MutationState.java

```diff
@@ -837,14 +838,18 @@ public long getBatchCount() {
         private final PName hTableName;
         @Nonnull
         private final TableRef origTableRef;
+        //This is a phoenix/logical table name
+        private String pTableFullName;
```

Comment:
Should not this be of type PName instead of String in this context to be compatible with the rest of the implementation?

Reply:
Should be fine, this class is only used in this file. Didn't involve any broader implications.

Reply:
The question is then why is hTableName of type PName whereas pTableFullName is of type String? Please note that PName is designed to be used as the name type for objects in Phoenix (Phoenix Name for short). So, if you do not have any specific reason to use String then you should use PName, especially here for consistency. The type for pTableFullName is visible via the constructor method, so it is not internal to this class.

Reply:
Okay.